<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>302</width>
    <height>259</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>302</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Z64 Compression/Decompression GUI</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="openFileInBtn">
       <property name="text">
        <string>File In</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="fileInLineEdit">
       <property name="text">
        <string>/home/github/z64decompress/dec_ntsc10j.z64</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="openFileOutBtn">
       <property name="text">
        <string>File Out</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="fileOutLineEdit">
       <property name="text">
        <string>/home/github/Z64CompDecGUI/test.z64</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QComboBox" name="fileSizeTargetList"/>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QComboBox" name="codecList"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QComboBox" name="gameList"/>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="openCompressExeBtn">
       <property name="text">
        <string>Executable</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="openCompressExeLineEdit">
       <property name="text">
        <string>/home/github/z64compress/z64compress</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="compressBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Compress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delCacheBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Cache</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
